README.md

```diff
@@ -1,115 +1,9 @@
-## OpenShift 4.4+ Designs
-Access design documentation for features slotted in the 4.4 release of OpenShift. Click links below for specific section details.
+# OpenShift Designs
```

Comment:
Could this say OpenShift Console designs?  Then we could remove the word "designs" from the bullets below

Reply:
yup!